### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> FlightDetails.json]: Alt as 'presentation' or 'null' is not defined for decorative image after 'SFO' text.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7829]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ windows+ enter key.
3. Navigate to the 'FlightDetails.json' through tab key and activate it.
4. Now navigate to the image after 'SFO' text through up/down arrow key.
5. Observe the issue.

**Actual Result:**
For decorative image which is present after 'SFO' text, Alt as presentation or null is not defined.

**Expected Result:**
For decorative image which is present after 'SFO' text, Alt as presentation or null should be defined.
Screen reader focus should not land on the decorative image.
**User Impact:**
Screen reader user will not be able to understand that whether the image is decorative or informative due to which user will get confused and will have difficulty in navigating or in tracking the content.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.01.01

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/104483005/189847285-1c58e1b5-f65d-4a0d-ae1a-f32ad5f5496c.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
As there is no decorative image is getting displayed, hence closing this issue.

![image](https://github.com/user-attachments/assets/aff76c2d-dcd7-4db0-bcac-079bc2842457)
